src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -75,49 +78,65 @@ export const RegistrationFormActions = ({
         }}>
         {tabNumber > RegistrationTab.Description && (
           <Box sx={{ gridArea: 'back-button' }}>
-            <Button
-              variant="outlined"
-              data-testid="button-previous-tab"
-              startIcon={<ArrowBackIcon />}
-              onClick={() => setTabNumber(tabNumber - 1)}>
-              {tabNumber === RegistrationTab.ResourceType && t('registration.heading.description')}
-              {tabNumber === RegistrationTab.Contributors && t('registration.heading.resource_type')}
-              {tabNumber === RegistrationTab.FilesAndLicenses && t('registration.heading.contributors')}
-            </Button>
+            <Tooltip title={t('common.previous')}>
+              <IconButton onClick={() => setTabNumber(tabNumber - 1)} disabled={isFirstTab}>
```

Comment:
Denne blir vel alltid `false` nå, pga sjekken på linje 79?
```suggestion
              <IconButton onClick={() => setTabNumber(tabNumber - 1)}>
```